frontend/src/components/elements/ImageList/ImageList.test.tsx

```diff
@@ -93,36 +90,15 @@ describe("ImageList Element", () => {
     })
   })
 
-  it("should render absolute src", () => {
```

Comment:
This is no longer necessary (we only care that we call `StreamlitEndpoints.buildMediaURL` - we don't care how the resulting URL is constructed).